packages/elements/src/internals/machines/sign-in/verification.machine.ts

```diff
@@ -145,6 +145,7 @@ const SignInVerificationMachine = setup({
   },
   guards: {
     isResendable: ({ context }) => context.resendable || context.resendableAfter === 0,
+    isNeverResendable: ({ context }) => context.currentFactor?.strategy === 'password',
```

Comment:
Arent't there more strategies that are never resendable?
For example: `passkey`, `google_one_tap`, `saml`

Reply:
There are, though I wasn't looking to address each of them in this PR. Especially, as `passkey` and `google_one_tap` haven't been integrated yet. While retaining the other flow isn't ideal, it doesn't cause any issues either.